src/EnergyPlus/HVACVariableRefrigerantFlow.hh

```diff
@@ -297,12 +297,14 @@ namespace HVACVariableRefrigerantFlow {
         bool EMSOverrideHPOperatingMode;
         Real64 EMSValueForHPOperatingMode;
         int HPOperatingModeErrorIndex;
-        Real64 VRFHeatRec;       // Heat Recovery heat reclaim power (W)
-        Real64 VRFHeatEnergyRec; // Heat Recovery heat reclain energy (J)
-        int HeatCapFTErrorIndex; // warning message index
-        int CoolCapFTErrorIndex; // warning message index
-                                 // The following are for the Algorithm Type: VRF model based on physics, applicable for Fluid Temperature Control
-        int AlgorithmIUCtrl;     // VRF indoor unit contrl algorithm, 1-High sensible, 2-Te/Tc constant
+        Real64 VRFHeatRec;         // Heat Recovery heat reclaim power (W)
+        Real64 VRFHeatEnergyRec;   // Heat Recovery heat reclain energy (J)
+        int HeatCapFTErrorIndex;   // warning message index
+        int CoolCapFTErrorIndex;   // warning message index
+        int HeatEIRFPLRErrorIndex; // warning message index
+        int CoolEIRFPLRErrorIndex; // warning message index
```

Comment:
The new trend is to initialize right here, that way you don't have to change the constructor. Mixing initializations is allowed (from @Myoldmopar).

int HeatCapFTErrorIndex = 0;

Reply:
Thanks! Changed to this way for the error index variables that appeared in the current struct.